okhttp/src/main/java/okhttp3/internal/ws/RealWebSocket.kt

```diff
@@ -63,7 +63,7 @@ class RealWebSocket(
   private var call: Call? = null
 
   /** This task processes the outgoing queues. Call [runWriter] to after enqueueing. */
-  private val writerTask: Task
+  private var writerTask: Task? = null
```

Comment:
lateinit? or worried about allocation costs?

Reply:
actually I believe lateinit is free! This is mostly for internal consistency with the code around it, that uses nullable where lateinit is probably an upgrade.

Lemme save for follow-up unless somebody else beats me to it